assets/src/edit-story/components/library/panes/shared/libraryMoveable.js

```diff
@@ -83,12 +83,25 @@ function LibraryMoveable({
   const { backgroundElement } = useStory((state) => ({
     backgroundElement: state.state.currentPage?.elements?.[0] ?? {},
   }));
-  const { fullbleedContainer, nodesById, pageContainer } = useCanvas(
-    (state) => ({
-      fullbleedContainer: state.state.fullbleedContainer,
-      pageContainer: state.state.pageContainer,
-      nodesById: state.state.nodesById,
-    })
+  const {
+    fullbleedContainer,
+    nodesById,
+    pageContainer,
+    designSpaceGuideline,
+  } = useCanvas((state) => ({
+    fullbleedContainer: state.state.fullbleedContainer,
+    pageContainer: state.state.pageContainer,
+    nodesById: state.state.nodesById,
+    designSpaceGuideline: state.state.designSpaceGuideline,
+  }));
+
+  const toggleDesignSpace = useCallback(
+    (visible) => {
+      if (designSpaceGuideline) {
+        designSpaceGuideline.style.visibility = visible ? 'visible' : null;
+      }
```

Comment:
Doesn't this clash with the class stuff? Or does it combine nicely always?

Reply:
Setting it to `null` makes it work together with the class.